definitions/variable/industry/tag_chemical_commodities.yaml

```diff
@@ -0,0 +1,46 @@
+- Chemicals Commodity:
+    - Ammonia:
+        description: ammonia (NH3), precursor chemical for nitrogen fertilisers
+        tier: "1"
```

Comment:
This is still using an outdated approach for tier-assigment, please remove or use the new approach.

- In the variable, assign a tier as an integer.
- In the tag list, you can use `^1` or `^2` to increase the tier-number for that tag-item.

So in this case, if you have

```yaml
- Demand|Chemicals|{Chemicals Commodity}:
     description: Demand of {Chemicals Commodity}
     unit: Mt/yr
     tier: 2
```
and a tag-list:
```yaml
- Chemicals Commodity:
     - Ammonia:
         description: ammonia (NH3), precursor chemical for nitrogen fertilisers
   - Nitrogen Fertilizer:
         description: single-nutrient nitrogen fertilizers and nitrogen containing
           multi-nutrient fertilizers
         tier: ^1
```
the demand for ammonia has tier 2 and demand for nitrogen fertilizer is tier 3.

Please do not use tier 1 unless it is mandatory for models to report this variable in all PRISMA scenarios.